### PR TITLE
Add i18n json schema

### PR DESCRIPTION
**Pull request by eyurko**

### CODE REVIEW

**Comment by eyurko** [comment]
Whoops -- let's try that again. I think I signed up via the appropriate channels, this time.

File: schema/i18n.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
+"interview_question": {
+  "title": "Internationalization (i18n)",
+  "prompts": [
+    {
+      "instructions": "Welcome to this programming exercise.\n\nYou will complete this exercise by editing the code in the sample.py file\nusing your favorite editor.\nEvery time you save the file, I will evaluate your code and let you know about\nnext steps.\n\nYour first task is to implement the abbreviate function. It takes a string as input\nand return an abbreviation of the string of the form <first character><length of the\nmiddle of the string><last character>. For example, \"internationalization\" should be\nabbreviated as \"i18n\".",
```
Comment (seanlip) [inline schema/i18n.json]: Do you need the intermediate newlines, e.g. between 'file' and 'using'? I thought we could just trust the UI to handle wrapping, and you'd only need paragraph breaks.

Speaking of paragraph breaks, if you want those too, would it be better to have a list of strings?

File: schema/i18n.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      "prerequisite_skills": ["Arrays", "Strings", "String Manipulation"],
+      "acquired_skills": ["String Manipulation"],
+      "input_function": null,
+      "output_function": null,
+      "main_function":, sample.abbreviate,
```
Comment (seanlip) [inline schema/i18n.json]: Might need to make the values strings, otherwise the JSON is invalid.

File: schema/i18n.json [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      ],
+      "buggy_outputs": [],
+      "performance_test_data": {
+        "input_data_atom": "o",
+        "transformation_function": createListOfUniqueStrings,
+        "expected_performance": "linear",
+        "evaluation_function": sample.all_are_unique,
+      },
+    },
+  ]
```
Comment (seanlip) [inline schema/i18n.json]: Is there a missing comma here?

You might want to plug this into an online json linter/validator.

File: schema/i18n.json (hunk 1 above)
Comment (eyurko) [inline schema/i18n.json]: I was just copying the format from the previous instance, but I can make these lists of strings if you'd like.

**Comment by eyurko** [comment]
I'm going to submit a PR from a different branch -- want to put this in my experimental directory so I can use cider.

**Pull request closed without merging**